src/deployers/modulus.js

```diff
@@ -48,6 +48,17 @@ exports.init = function(options) {
     });
   };
 
+  var setEnvironmentVariables = function(callback) {
+    if (options.env) {
```

Comment:
Instead of having all the interesting code indented in an `if`, it's often cleaner, and easier to follow, to return early: `if (!options.env) return callback();`